scratch/tmpl.Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM scratch
 COPY certs/ca-certificates.crt /etc/ssl/certs/
 COPY traefik /
+WORKDIR /tmp
```

Comment:
These two `WORKDIR` should be before the first `COPY` we will be able to use cache ;)